site/schemas/login.schema.ts

```diff
@@ -0,0 +1,9 @@
+import { z } from "zod";
+import { setZodDefaultError } from "../utils";
+
+export const loginSchema = z.object({
+    email: z.string(setZodDefaultError("Enter an email address")).min(1),
+    password: z.string(setZodDefaultError("Enter a password")).min(8),
```

Comment:
I'd probably just make the min 1 here, if the user accidentally misses a character off their password they might be confused if they get "Enter a password" as the error

Reply:
ive updated the code to use min as 1